packages/appstore/src/components/modals/real-wallets-upgrade/components/content-with-link/content-with-link.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import classNames from 'classnames';
+import './content-with-link.scss';
+
+type TContentWithLinkProps = {
+    children: JSX.Element | Array<JSX.Element>;
+    is_mobile?: boolean;
+    is_right_forked?: boolean;
+    has_fork?: boolean;
```

Comment:
Just a question but is there an example of a use case where `ContentWithLink` has no fork? Based on the explanation it looks like this component would either have a left fork or a right fork, would the `has_fork` boolean then be needed?

Reply:
Hi @lubega-deriv,
The link has to sections, the straight neck and the fork part as show below. `has_fork` can be used to show/hide the fork part but the link will always contain the neck (the neck resizes dependent on whether the fork is present or not). So we do need the prop 🙂.

<img width="150" alt="image" src="https://github.com/binary-com/deriv-app/assets/125039206/c9a41c95-5dca-4be9-b2b0-438f2755b4ea">

Reply:
I see.. alright thanks for the clarification!

Reply:
This is not applicable any more.